src/snippets.ts

```diff
@@ -0,0 +1,83 @@
+import {ElementSnippet} from './editor-service';
+
+const snippets: ElementSnippet[] = [
+  {
+    'tagname': 'html-import',
+    'description': 'Template definition of an HTML import.',
+    'prefix': 'hi',
+    'expandTo': '<link rel="import" href="bower_components/.html">',
+    'expandToSnippet': '<link rel="import" href="${1:bower_components}/${2}/${2}.html">'
+  },
+  {
+    'tagname': 'polymer-element',
+    'description': 'Template definition of a Polymer 2 element',
+    'prefix': 'pe',
+    'expandTo': '',
+    'expandToSnippet': `
+<link rel="import" href="bower_components/polymer/polymer-element.html">
+
+<dom-module id="\${2}">
+  <template>
+    <custom-style>
+      <style>
+        :host {
+          display: block;
+        }
+      </style>
+    </custom-style>
+  </template>
+  <script>
+    class \${1} extends Polymer.Element {
+      static get is() {return '\${2}';}
+      static get config() {
+        return {
+          properties: {
+            $3
+          }
+        };
+      }$0
+    }
+    customElements.define(\${1}.is, \${1});
+  </script>
+</dom-module>
+`
+  },
+  {
+    'tagname': 'tdd-test',
+    'description': 'Template definition for a TDD test suite',
+    'prefix': 'tdd-test',
+    'expandTo': '',
+    'expandToSnippet': `
+<html>
+  <head>
+    <meta name="viewport" content="width=device-width, minimum-scale=1.0, initial-scale=1.0, user-scalable=yes">
+    <title>$1</title>
+    <script src="\${2:bower_components}/webcomponentsjs/webcomponents-lite.js"></script>
```

Comment:
I think a lot of this could be based off the element tagname. i.e.


```html
<title>${1:my-element}</title>
...
<link rel="import" href="./$1.html">
...
<test-fixture id="basic">
  <template>
    <$1></$1>
  </template>
</test-fixture>
...
<script>
  suite('$1 tests', function() {
    ...
  });
</script>